src/lib/viewers/controls/color-picker/ColorPickerToggle.scss

```diff
@@ -0,0 +1,59 @@
+@import '~box-ui-elements/es/styles/variables';
+@import '../styles';
+
+.bp-ColorPickerToggle {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 48px;
+    height: 48px;
+
+    .bp-ColorPickerToggle-button {
```

Comment:
The SUIT naming convention allows us to avoid nesting rules and increasing specificity. Let's flatten this file out a bit.

Reply:
Just to clarify, for this scenario, do you mean moving `.bp-ColorPickerToggle-button` to be on the same level as `.bp-ColorPickerToggle`?

For example:
```
.bp-ColorPickerToggle { ... }

.bp-ColorPickerToggle-button { ... }
```

I'm looking at the example of FeedCard.scss in https://confluence.inside-box.net/display/ETO/CSS+Class+Naming and it looks like that's how the flattening works.

Reply:
Yep!